README.md

```diff
@@ -188,8 +183,8 @@ I would like to thank Microverse and all the persons that got in touch with me w
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT] (./LICENSE) licensed.
+<br>
+This projectt is Behance (https://creativecommons.org/licenses/by-nc/4.0/legalcode) Licensed.
```

Comment:
- [ ] It is required to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly do that in your README file.